src/hosted.js

```diff
@@ -93,8 +93,14 @@
     }
 
     function load(Bramble) {
+        var hash = location.hash.replace(/^#/, "");
```

Comment:
In the issue you file from the comment above, can you link to this piece of code as well so that we don't forget to change it as well when we move to the bramble api.